mmcv/ops/csrc/common/pytorch_cpp_helper.hpp

```diff
@@ -21,4 +22,10 @@ using namespace at;
   CHECK_CPU(x);            \
   CHECK_CONTIGUOUS(x)
 
+inline int GET_BLOCKS(const int N, const int num_threads = THREADS_PER_BLOCK) {
```

Comment:
Why do we put `THREAD_PER_BLOCK` and `GET_BLOCKS` here since we already have them in common_cuda_helper.hpp?

Reply:
Because `iou3d.cpp` needed it. But according to the above discussion, maybe `iou3d.cpp` should not use `GET_BLOCKS`, so this `THREAD_PER_BLOCK` and `GET_BLOCKS` will be removed.